beacon_node/execution_layer/src/lib.rs

```diff
@@ -1788,14 +1787,18 @@ impl<T: EthSpec> ExecutionLayer<T> {
             return Ok(None);
         };
 
-        let convert_transactions = |transactions: Vec<EthersTransaction>| {
+        let convert_transactions = |transactions: Vec<TransactionSigned>| {
             VariableList::new(
                 transactions
                     .into_iter()
-                    .map(ethers_tx_to_ssz::<T>)
-                    .collect::<Result<Vec<_>, BlobTxConversionError>>()?,
+                    .map(|tx| {
+                        let mut buf = vec![];
```

Comment:
Not super important and not sure if it makes much of a difference, but we could maybe reduce some resizings by doing
```rust
let mut buf = Vec::with_capacity(min(tx.size(), SOME_LARGE_NUMBER));
```

Reply:
good shout!